relayer/chains/sui/tx.go

```diff
@@ -3,30 +3,31 @@ package sui
 import (
 	"context"
 
-	"github.com/icon-project/centralized-relay/relayer/types"
+	relayerTypes "github.com/icon-project/centralized-relay/relayer/types"
 )
 
-func (p Provider) QueryTransactionReceipt(ctx context.Context, txDigest string) (*types.Receipt, error) {
-	//Todo
-	return nil, nil
+func (p *Provider) QueryTransactionReceipt(ctx context.Context, txDigest string) (*relayerTypes.Receipt, error) {
+	txBlock, err := p.client.GetTransaction(ctx, txDigest)
+	if err != nil {
+		return nil, err
+	}
+	receipt := &relayerTypes.Receipt{
+		TxHash: txDigest,
+		Height: txBlock.TimestampMs.Uint64(),
```

Comment:
I should be `Height: txBlock.Checkpoint.Uint64()`

Reply:
fixed.